kie-wb-common-react/boxed-expression-editor/__mocks__/test-setup.js

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const enzyme = require('enzyme');
```

Comment:
I've noticed that `enzyme` seems to be removed from `kogito-tooling` (https://github.com/kiegroup/kogito-tooling/pull/220/files). Shouldn't we do the same? :-)

Reply:
Ok, I will try to remove it from this project too. Will have a dedicated commit in the next PR.